app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class=sign_in>
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field_1">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+  
+  <div class="field_1">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %><br />
+  </div>
+
+  <div class="field_1">
+    <%= f.label :photo %><br />
+    <%= f.text_field :photo %><br />
+  </div>
```

Comment:
- [ ] I am unable to register a new user, check the attached image below to see the error it outputs when I try to register a new user. Note that this has hindered me to review other expected functionalities and the next reviewer will be looking for these functionalities:
    - A user should be able to log in with a combination of email and password.
    - A user should be able to reset the password in the application.


<img width="610" alt="Screen Shot 2021-12-08 at 5 40 19 PM" src="https://user-images.githubusercontent.com/47994600/145257245-3cd9294f-308f-4548-9d83-b67004187a0c.png">